app/scripts/modules/netflix/ci/ci.dataSource.js

```diff
@@ -1,20 +1,21 @@
 import {DataSourceConfig} from 'core/application/service/applicationDataSource';
 import {APPLICATION_DATA_SOURCE_REGISTRY} from 'core/application/service/applicationDataSource.registry';
 import {NetflixSettings} from '../netflix.settings';
+import {CI_BUILD_READ_SERVICE} from './services/build.read.service';
```

Comment:
If we're going to rename modules/components, let's rename the files, too. I would expect `build.read.service` to include a class named `BuildReader`, not `CiBuildReader`.